src/runtime/MethodBinder.cs

```diff
@@ -28,17 +28,22 @@ internal class MethodBinder
 
         [NonSerialized]
         public bool init = false;
+
         public const bool DefaultAllowThreads = true;
         public bool allow_threads = DefaultAllowThreads;
 
-        internal MethodBinder()
+        public bool argsReversed = false;
+
+        internal MethodBinder(bool argsReversed = false)
         {
             list = new List<MaybeMethodBase>();
+            this.argsReversed = argsReversed;
         }
 
-        internal MethodBinder(MethodInfo mi)
+        internal MethodBinder(MethodInfo mi, bool argsReversed = false)
```

Comment:
IMHO, you don't need this parameter in constructor. Just set the field directly when needed.

E.g. `new MethodBinder(...) { argsReversed = true }`